VERSION

```diff
@@ -1 +1 @@
-0.1.0
+0.1.5
```

Comment:
We should increment the minor version instead of patch version due to the nature of this feature in our case. Thoughts?

Reply:
That makes sense, I'll update everything soon.